ajax/libs/imgcache.js/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "imgcache.js",
+  "version": "2.0.0",
+  "description": "JS library based on the File API to cache images for offline recovery (target: cordova/phonegap & chrome)",
+  "filename": "imgcache.min.js",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/chrisben/imgcache.js.git"
+  },
+  "keywords": [
+    "image",
+    "cache",
+    "html5",
+    "file",
+    "filesystem",
+    "offline"
+  ],
+  "author": "Christophe BENOIT",
+  "license": "Apache-2.0",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/chrisben/imgcache.js.git",
+    "fileMap": [
+      {
+        "basePath": "js",
+        "files": [
+          "*imgcache*.js"
+        ]
+      },
+      {
+        "basePath": "lib",
+        "files": [
+          "*imgcache*.js"
```

Comment:
We can just use `*imgcache*` or `*imgcache*.+(css|map|js)` pattern to get the files.
If the lib be release the *.css, we can get them by this pattern.
Thanks